pkg/protoconv/time.go

```diff
@@ -11,11 +11,16 @@ import (
 
 var (
 	log = logging.LoggerForModule()
-)
 
-const (
-	timeFormat         = "2006-01-02T15:04Z"
-	extendedTimeFormat = "2006-01-02T15:04:03Z"
+	// timeLayouts lists each known time format returned by vulnerability scanners.
```

Comment:
nit: since this package isn't used only for vuln scanners, don't want folks to be 'detered' from adding formats for other purposes. 

```suggestion
	// timeLayouts lists each known time format used throughout StackRox
```

Reply:
true, yet the godoc above `ConvertTimeString` where this is used does explicitly call out vulnerabilities. I'll make the change, though

Reply:
Ohh, I didn't notice that note in the `ConvertTimeString` godoc, based on that - a super nit: rename the method to `ConvertVulnTimeString` 😉

Reply:
eh I'll keep it as-is since it's used in migration tests